generator/explores/explore.py

```diff
@@ -25,6 +25,19 @@ def to_dict(self) -> dict:
 
     def to_lookml(self) -> dict:
         """Generate LookML for this explore."""
+        base_lookml = {}
+        for view_type, view in self.views.items():
+            if "join" in view_type:
+                continue
+            if self._get_view_has_submission(view):
+                base_lookml[
+                    "sql_always_where"
+                ] = f"${{{self.name}.submission_date}} >= '2010-01-01'"
+
+        base_lookml.update(self._to_lookml())
```

Comment:
Is there a reason why you're updating this here, and not just saying `base_lookml = self._to_lookml()` on top? Do you want `sql_always_where` to be overriden by the definition? If so I'd try to make this explicit, maybe with some comments (as suggested just above)

Reply:
Indeed, the `_to_lookml` should take precedence. I'll add a comment